File: Assignment/Module-1(fundamentals/p8.cpp
#include<stdio.h>
main()
{
	float English,Maths,Science,Chemistry,Physics;
	float totalmarks,percentage;
	printf("Enter marks for English");
	scanf("%f",&English);
	printf("Enter marks for Maths");
	scanf("%f",&Maths);
	printf("Enter marks for Science");
	scanf("%f",&Science);
	printf("Enter marks for Chemistry");
	scanf("%f",&Chemistry);
	printf("Enter marks for Physics");
	scanf("%f",&Physics);
	totalmarks=English+Maths+Science+Chemistry+Physics;
	percentage=(totalmarks/500)*100;
	printf("total marks: %.2f\n",totalmarks);
	printf("percentage: %.2f%%\n",percentage);
}

File: Assignment/flutter dart assignment module-1/p8.dart
import 'dart:io';

void main() {
  double subject1, subject2, subject3, subject4, subject5;
  
  stdout.write("Enter marks for subject 1: ");
  subject1 = double.parse(stdin.readLineSync()!);

  stdout.write("Enter marks for subject 2: ");
  subject2 = double.parse(stdin.readLineSync()!);

  stdout.write("Enter marks for subject 3: ");
  subject3 = double.parse(stdin.readLineSync()!);

  stdout.write("Enter marks for subject 4: ");
  subject4 = double.parse(stdin.readLineSync()!);

  stdout.write("Enter marks for subject 5: ");
  subject5 = double.parse(stdin.readLineSync()!);

  double totalMarks = subject1 + subject2 + subject3 + subject4 + subject5;
  
  double percentage = (totalMarks / 500) * 100;

  print("Total Marks: $totalMarks");
  print("Percentage: $percentage%");
}
